### PR TITLE
Facebook homepage posts

### PR DESCRIPTION
**Pull request by Guilhemfuel**

### CODE REVIEW

File: api/lumen/app/Http/Controllers/SocialController.php [hunk 1]
```diff
@@ -277,15 +277,21 @@ public function facebookFeed()
       $posts = json_decode($json, true);
       $posts = $posts['data'];
 
+      if (empty($client_id) || empty($client_secret)) {
```
Comment (Bramvanosta) [inline api/lumen/app/Http/Controllers/SocialController.php]: Missing a test if the returned string is empty
